tests/recipe/DELETE_recipes.test.js

```diff
@@ -0,0 +1,63 @@
+const supertest = require("supertest");
+const app = require("../../app");
+const Recipe = require("../../models/recipe");
+const User = require("../../models/user");
+const api = supertest(app);
+const mongoose = require("mongoose");
+
+const { initialRecipes, initialUsers, recipesInDb } = require("../test_helper");
+const STATUS_CODE = require("../../shared/errorCode");
+
+const { createUserAndLogin, addRecipe } = require("../api_test_helpers.js");
+
+const RECIPE_API = "api/recipe";
+const DELETE = "delete";
+
+describe("recipe api", () => {
+  let token, authHeader, userId, recipeId;
```

Comment:
it's still global in way, because it's in scope of all the tests. ideally, each test should be isolated.